eng/pipelines/checkout-unix-task.yml

```diff
@@ -0,0 +1,11 @@
+# Shallow checkout sources on Unix
+steps:
+  - checkout: none
+
+  - script: |
+      set -x
```

Comment:
only difference is how to get nice echoing of the git commands. But I felt like it was worth copy/pasting for. not strongly attached to it.

Reply:
As a follow up, I would like to see if we could have an invalid command in the script and share a single yml? 

```
  - script: |
      # One of these will fail, but no biggie. Could redirect (2>NUL) output to make it less noisy
      set -x
      @echo on
```